gno.land/pkg/sdk/vm/msgs.go

```diff
@@ -11,6 +11,15 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/std"
 )
 
+type Format string
+
+const (
+	FormatMachine = "machine" // Default machine representation
+	FormatJSON    = "json"
```

Comment:
Maybe let's add a comment like `// XXX: EXPERIMENTAL, only supports primitive types for now`

Reply:
e906187c0